feature.js/index.js

```diff
@@ -0,0 +1,62 @@
+import {
+  CookieInstrument,
+  JavascriptInstrument,
+  HttpInstrument,
+  NavigationInstrument,
+} from "openwpm-webext-instrumentation";
+
+import * as loggingDB from "./loggingdb.js";
+
+async function foo() {
```

Comment:
can we call this `main` or something similar? `foo` is typically used for example code.